### PR TITLE
Test startup.cs and swagger

### LINKED ISSUE #271: Component test

### PR DESCRIPTION
**Pull request by Christdej**
Closes #271

### CODE REVIEW

File: backend/api/appsettings.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  },
-  "ApplicationInsights": {
-    "InstrumentationKey": ""
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+    "AzureAd": {
```
Comment (ReedOnly) [inline backend/api/appsettings.json:2]: whats wrong with this file?

File: backend/tests/StartupTest.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public class ComponentTest
+    {
+        private readonly WebApplicationFactory<Startup> factory = new WebApplicationFactory<Startup>();
+
+        [Fact]
+        public async Task StartupTest()
+        {
+            var client = factory.CreateClient();
+
+            var response = await client.GetAsync("/swagger/index.html");
```
Comment (ReedOnly) [inline backend/tests/StartupTest.cs:19]: This only checks if swagger exists, not if API works right?

**Review by ReedOnly — APPROVED** [review]

File: backend/api/appsettings.json (hunk 1 above)
Comment (Christdej) [reply backend/api/appsettings.json:2]: Changing from 2 spaces to 4, because I like 4 spaces better

File: backend/tests/StartupTest.cs (hunk 2 above)
Comment (Christdej) [reply backend/tests/StartupTest.cs:19]: This checks startup.cs, and that swagger returns 200-299 code. If you mess something up in startup.cs, this test will fail

**Pull request merged**